### ISSUE TITLE
Support feeds with incorrect content-type headers

### ISSUE THREAD

**Issue by romlok** [gpodder/gpodder#225]
As with the poster of bug #224, I have found a feed I'd like to subscribe to, where the content-type of the served RSS feed is incorrectly provided as `text/html; charset=UTF-8`, so gPodder assumes it's an HTML page and refuses to add the feed: 

> Some podcasts could not be added to your list:
> 
> http://boilingsteam.com/feed/podcast/boilingsteam-regular: Got HTML document instead

I've sent a message to the owners of the website asking them to fix it, but misconfigured webservers are not an uncommon sight in general, so gPodder should probably just try to parse the document regardless of the content-type header.

**Comment by zajc** [comment]
I have exactly the same problem in version 3.9.3. I cannot retrieve feeds because the content-type header is not xml. The feed is http://podcast.rtvslo.si/frekvenca_x.xml.

```
curl -I http://podcast.rtvslo.si/frekvenca_x.xml
HTTP/1.1 200 OK
Server: nginx
Date: Mon, 20 Mar 2017 14:31:33 GMT
Vary: Accept-Encoding
Vary: Accept-Encoding
Set-Cookie: dancer.session=22355439973059243799676143036067884; path=/; expires=Mon, 20-Mar-2017 22:37:53 GMT; HttpOnly
X-Powered-By: Perl Dancer 1.3129
Content-Length: 203512
Content-Type: text/html; charset=utf-8
Via: 1.1 ironport5.pop-tv.si:80 (Cisco-WSA/9.2.0-809)
Connection: keep-alive
```
I also think gpodder should just try to parse the document regardless of the content-type header.

**Comment by G1tOnW1th1t** [comment]
I'd also like to see a fix for this please as I have no idea how to add a podcast feed that has just come up with this error.

**Comment by elelay** [comment]
This is fixed in gPodder 3.9.6 and 3.10.0
